client/webserver/locales/en-us.go

```diff
@@ -131,8 +131,7 @@ var EnUS = map[string]string{
 	"Set App Password":          "Set App Password",
 	"reg_set_app_pw_msg":        "Set your app password. This password will protect your DEX account keys and connected wallets.",
 	"Password Again":            "Password Again",
-	"reg_dcr_required":          "Your Decred wallet is required to pay registration fees.", // TODO
-	"reg_dcr_unlocked":          "Unlock your Decred wallet to pay registration fees.",      // TODO
+	"reg_dcr_required":          "A wallet is required to pay registration fees. You can set one with this form.",
```

Comment:
The key name and text I think should change.

```suggestion
	"reg_wallet_required":       "A wallet is required to pay registration fees.",
```

Although I'm thinking this text is not even required on the form at all since the user just got here from the register form by clicking the Setup button, so they know why they are on this form.

Reply:
Since we went with getting rid of this as an argument to the `"newWalletForm"` template, it's the right time to clean up that template definition because it no longer gets an input from any callers and the template only has one possible rendering now.

That is, when called like `{{template "newWalletForm" "reg reg reg"}}`, we see:

![image](https://user-images.githubusercontent.com/9373513/135516599-4b6726d1-ba53-48f9-9213-d1b136acae4c.png)

When called like `{{template "newWalletForm"}}`, as it is now:

![image](https://user-images.githubusercontent.com/9373513/135516649-e46c5314-0747-49c0-b270-c589595e870e.png)

The two differences are obviously this text at the top, but also the text about the wallet password goes into a tooltip:

![image](https://user-images.githubusercontent.com/9373513/135516813-737040c7-eba2-4b44-8f34-b443c685ae9e.png)

So in forms.html where we have `{{define "newWalletForm"}}`, there are a bunch of `if`s using `.` that will never be true anymore since only this register form was ever executing this template with an arg.

Regarding the wallet password texts, we could just go with the existing `w_password_tooltip`, or combine it with the text from `w_password_helper`.  i.e.  `"w_password_tooltip":             "This is the password you have configured with your wallet software. Leave the password empty if there is no password required for the wallet.",`

![image](https://user-images.githubusercontent.com/9373513/135517971-88666fa3-85d0-444a-a6f1-f6698ff89c69.png)